logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserModel.kt

```diff
@@ -159,7 +159,8 @@ data class OtherUser(
     val botService: BotService?,
     val deleted: Boolean,
     val defederated: Boolean,
-    override val expiresAt: Instant? = null
+    override val expiresAt: Instant? = null,
+    val oneOnOneConversationId: ConversationId? = null
```

Comment:
suggestion: rename it to add an `active` prefix.

Seeing the `ConversationRepository` taking a `Protocol` parameter makes it clear that we can have more than one 1:1 between two users (one for each protocol). Which one would show up in this field?

```suggestion
    val activeOneOnOneConversationId: ConversationId? = null
```